go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/envoyproxy/go-control-plane
 
-go 1.17
+go 1.19
```

Comment:
Can we make this change in a separate PR, since it affects downstream projects?

Reply:
Yup I actually just opened this:
https://github.com/envoyproxy/go-control-plane/pull/714

Reply:
Reverted back to 1.17 for this PR so we can get it in before the multi-module release.